templates/infra.json

```diff
@@ -8,11 +8,11 @@
                 "description": "The name of the function app that you wish to create."
             }
         },
-        "engineContributorGUID": {
+        "engineReaderGUID": {
             "type": "string",
             "defaultValue": "[newGuid()]",
             "metadata": {
-                "description": "GUID for engine contributor role on subscription."
+                "description": "GUID for engine reader role on subscription."
```

Comment:
If we execute the ARM template multiple times, we'll have a new role assignment each time since we're generating a new guid. Couldn't we generate a guid based on a fixed value, e.g. resource group name or something?

Reply:
Decided to leave this and create a separate issue for the GUID